src/components/InfoTable/InfoTable.tsx

```diff
@@ -10,9 +10,14 @@ import {
   Box,
 } from '@mui/material';
 
```

Comment:
I split this type out so that I could type the return value of my formatter function that takes in `policyHolder` data and returns an array of formatted `TInfoTableRow`